python/sglang/bench_serving.py

```diff
@@ -222,6 +222,85 @@ async def async_request_openai_completions(
     return output
 
 
+async def async_request_truss(
```

Comment:
The reason for not directly using `async_request_openai_completions` here and writing a separate one instead is due to some differences in format, and this part will be changed later.